.circleci/config.yml

```diff
@@ -136,7 +136,48 @@ jobs:
                 tf-api-staging ${PROJECT_NAME}
             fi
 
-  deploy_production:
+  deploy_staging_eks:
+    <<: *defaults_py
+    steps:
+      - setup_remote_docker
+
+      - attach_workspace:
+          at: .
```

Comment:
why are you attaching the workspace?

Reply:
thanks.  to get the /kustomize directory, but i missed persisting that.  added.